python/cog/predictor.py

```diff
@@ -48,14 +49,15 @@
 
 log = structlog.get_logger("cog.server.predictor")
 
-ALLOWED_INPUT_TYPES: List[Type[Any]] = [
+ALLOWED_INPUT_TYPES: List[Union[Type[Any], Type[CommonChatSchemaChatMessage]]] = [
```

Comment:
I don't think this adds anything does it? `Any` covers `CommonChatSchemaChatMessage`, doesn't it?

Reply:
This solves the linting issue as seen here: https://github.com/replicate/cog/actions/runs/13705936291/job/38331101920

`UnionType` is apparently not covered by `Type[Any]`

Reply:
Wild. Ok!